packages/format-library/src/link/modal.native.js

```diff
@@ -130,19 +130,19 @@ class ModalLinkUI extends Component {
 				onClose={ this.onDismiss }
 				hideHeader
 			>
-				{ /* eslint-disable jsx-a11y/no-autofocus */
-					<BottomSheet.Cell
-						icon={ 'admin-links' }
-						label={ __( 'URL' ) }
-						value={ this.state.inputValue }
-						placeholder={ __( 'Add URL' ) }
-						autoCapitalize="none"
-						autoCorrect={ false }
-						keyboardType="url"
-						onChangeValue={ this.onChangeInputValue }
-						autoFocus={ Platform.OS === 'ios' }
-					/>
-				/* eslint-enable jsx-a11y/no-autofocus */ }
+				{ /* eslint-disable jsx-a11y/no-autofocus */ }
```

Comment:
Is it possible for this to be framed as an `// eslint-disable-next-line` ? Unsure if it's captured at the opening tag or at the prop assignment.

Reply:
Looks like it can't, unfortunately.